<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Inform</class>
 <widget class="QWidget" name="Form_Inform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::tab-bar{
        alignment:left;
}
QTabBar::tab{
	background:rgb(255, 255, 255, 100);
	color:#0068FF;
	min-width:12ex;
	min-height:5ex;
}
QTabBar::tab:hover{
	background:#0068FF;
    color:white;
}
QTabBar::tab:selected{
	border-color: white;
	background:#0068FF;
	color:white;
}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="SysInform">
    <attribute name="title">
     <string>系统信息</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>主控板软件版本：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FPGA软件版本：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>仪器序列号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_MachineSN">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>xxxxxxxxxxxx</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_MBSoftVersion">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>libs.xxxxx.Uxxxx.T1A.001</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_CHEVersion">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>341</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>libs.xxxxx.Uxxxx.T1A.001</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SystemState">
    <attribute name="title">
     <string>仪器状态</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>＋5VA电压：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_RefV">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>＋5VD电压：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>＋5V_DIS：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>＋24V电压：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>＋5V_E电压：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_5VE">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_5VDIS">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_5VD">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>REF_2.5V电压：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_24V">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_5VA">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>90</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>系统服务</string>
    </attribute>
    <widget class="QPushButton" name="btn_Calibrate">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Images/tslibadjust.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_SoftUpdate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Images/softupdate.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_SoftUpdate_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Images/softupdate.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>800</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/pagemenu.png);</string>
   </property>
   <widget class="QPushButton" name="btn_Esc1">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>11</y>
      <width>90</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/return.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
